man/systemd-stub.xml

```diff
@@ -133,8 +140,8 @@
     core kernel, the embedded initrd and kernel command line (see above for a full list).</para>
 
     <para>Also note that the Linux kernel will measure all initrds it receives into TPM PCR 9. This means
-    every type of initrd will be measured twice: the initrd embedded in the kernel image will be measured to
-    both PCR 4 and PCR 9; the initrd synthesized from credentials will be measured to both PCR 12 and PCR 9;
+    every type of initrd will be measured twice or thrice: the initrd embedded in the kernel image will be measured to
```

Comment:
While I am a great fan of thrice, it is rather uncommon in most English vernaculars, at least if the intention is not to sound like some ancient tome of forbidden knowledge. :)
```suggestion
    every type of initrd will be measured twice or three times: the initrd embedded in the kernel image will be measured to
```
or
```suggestion
    every type of initrd will be measured two or three times: the initrd embedded in the kernel image will be measured to
```